src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void userNotNull_Ok() {
+        User userTest = new User();
+        userTest.setId(31654L);
+        userTest.setLogin("Bond-1234");
+        userTest.setPassword("1234567");
+        userTest.setAge(43);
+        User register = registrationService.register(userTest);
+        assertNotNull(register.getLogin());
+    }
+
+    @Test
+    void passwordNotNull_Ok() {
+        User userTest = new User();
```

Comment:
This test is not testing any functionality of the registration service. Consider removing this test or refactoring it to test the registration service's behavior.